tests/fsharp/perf/tests_perf.fs

```diff
@@ -0,0 +1,35 @@
+module ``FSharp-Tests-Perf``
+
+open System
+open System.IO
+open NUnit.Framework
+
+open FSharpTestSuiteTypes
+open NUnitConf
+open PlatformHelpers
+
+let testContext = FSharpTestSuite.testContext
+
+
+module Graph = 
+
+    [<Test; FSharpSuitePermutations("perf/graph")>]
+    let graph p = check (processor {
+        let { Directory = dir; Config = cfg } = testContext ()
+        
+        do! SingleTestBuild.singleTestBuild cfg dir p
```

Comment:
for example, why isn't this 

    let ctxt = textContext()
    do! ctxt.singleTestBuild p

Seems reasonable?